dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheHashMap.java

```diff
@@ -205,7 +213,7 @@ protected Object loadOneValue(String attrName, BackedSession sess) {
             byte[] bytes = cacheStoreService.getCache(appName).get(key);
 
             if (trace && tc.isDebugEnabled())
-                Tr.debug(this, tc, key.toString(), bytes.length);
+                Tr.debug(this, tc, key.toString(), bytes == null ? null : bytes.length);
```

Comment:
perhaps add a message here indicating what the trace value is?  Something like:
`"byte length for session=" + key.toString()`